.pre-commit-config.yaml

```diff
@@ -59,12 +65,21 @@ repos:
   hooks:
   - id: docconvert
     args: ["numpy"]
-- repo: https://github.com/b8raoult/optional-dependencies-all
```

Comment:
since this being removed, should that also be mentioned in the Changelog?

Reply:
It's basically just changing how optional dependencies are built.